### DISCUSSION TITLE
Open source server libs?

### DISCUSSION THREAD

**Discussion by MentalGear** [liveblocks/liveblocks#1466] [Feedback]
I was wondering if liveblocks also plans to open-source their server-side packages? (not just clients)

**Comment by nvie** [comment]
We’re planning to open source a server package in the coming months, so stay tuned.

**Reply by taylor-shift** [reply]
Any updates on this?

**Reply by nyxb** [reply]
I would also be interested

**Comment by N0N1m3** [comment]
@GuillaumeSalles @nvie Any updates on this?
